### DISCUSSION TITLE
How do I create a signature and store in a database with c#.net

### DISCUSSION THREAD

**Discussion by dcisbackagain** [community/community#59573] [Programming Help]
### Select Topic Area

Question

### Body

Please I am creating a desktop clinic management system with signature integration and I am completely stuck with the signature integration. Any help?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dcisbackagain ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by queenofcorgis** [comment]
Thanks for replying, @SecHex ❤️, thanks for breaking down the steps. @dcisbackagain did this help you, if so don't forget to mark it as the answer in case other members have a similar ask ✅ . Thanks!
